packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.java

```diff
@@ -49,13 +48,14 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.UUID;
 
 public class ImagePickerModule extends ExportedModule implements ActivityEventListener {
 
   public static final String TAG = "ExponentImagePicker";
-  public static final String MISSING_ACTIVITY = "MISSING_ACTIVITY";
-  public static final String MISSING_ACTIVITY_MESSAGE = "Activity which was provided during module initialization is no longer available";
+  private static final String MISSING_ACTIVITY = "MISSING_ACTIVITY";
+  private static final String MISSING_ACTIVITY_MESSAGE = "Activity which was provided during module initialization is no longer available";
+  private static final String CAN_NOT_DEDUCE_TYPE = "ERR_CAN_NOT_DEDUCE_TYPE";
```

Comment:
Let's use `ERR` in the property name too

```suggestion
  private static final String ERR_CAN_NOT_DEDUCE_TYPE = "ERR_CAN_NOT_DEDUCE_TYPE";
```